### PR TITLE
Document AssetTargetFallback

### LINKED ISSUE #2816: [2.0] Describe AssetTargetFallback
Similar to the request to surface `AssetTargetFallback` in https://github.com/NuGet/docs.microsoft.com-nuget/issues/410, which was prompted by a discussion regarding `PackageTargetFallback` [here](https://github.com/dotnet/cli/issues/3439#issuecomment-319702662), there are a few topics over here that might (probably) need a smidgen of work ... a note? a section? a replacement? versioned content? ... something to surface it:

https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#packagetargetfallback
https://docs.microsoft.com/en-us/dotnet/core/tools/project-json-to-csproj#imports

Cross ref: [Documenting .NET Core MSBuild targets and properties/CSPROJ (2642)](https://github.com/dotnet/docs/issues/2642)

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #2816

### CODE REVIEW

**Review by rainersigwald — APPROVED** [review]
LGTM but @nkolev92's signoff counts 10x mine on this.

**Review by nkolev92 — APPROVED** [review]
Looks great. 

The docs for this are overdue. 

On NuGet side we are planning on adding some examples, maybe we can link those when we get to that ourselves.

File: docs/core/project-sdk/msbuild-props.md [hunk 1]
```diff
@@ -148,6 +149,21 @@ The `PackageReference` item lets you specify a NuGet dependency. For example, yo
...
+
+The `AssetTargetFallback` property lets you specify additional compatible framework versions for projects that your project references and NuGet packages that your project consumes. For example, if you specify a package dependency using `PackageReference` but that package doesn't contain assets that are compatible with your projects's `TargetFramework` version, the `AssetTargetFallback` property comes into play. The compatibility of the referenced package is rechecked using each framework version that's specified in `AssetTargetFallback`.
+
+You can set the `AssetTargetFallback` property to one or more [target framework versions](../../standard/frameworks.md#supported-target-framework-versions).
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  ...
+  <PropertyGroup>
+    <AssetTargetFallback>netstandard1.6</AssetTargetFallback>
```
Comment (gewarren) [inline docs/core/project-sdk/msbuild-props.md]:
```suggestion
    <AssetTargetFallback>net461</AssetTargetFallback>
```

**Comment by gewarren** [comment]
Thanks for the reviews Rainer and Nikolche.

**Pull request merged**